lib/fog/ovirt/requests/compute/v4/list_clusters.rb

```diff
@@ -3,8 +3,9 @@ module Compute
     class Ovirt
       class V4
         class Real
-          def list_clusters(filters = {})
-            client.system_service.clusters_service.list(filters).map { |ovirt_obj| ovirt_attrs ovirt_obj }
+          def list_clusters(opts = {})
+            opts[:search] += " datacenter=#{datacenter_hash[:name]}"
+            client.system_service.clusters_service.list(opts).map { |ovirt_obj| ovirt_attrs ovirt_obj }
```

Comment:
Nitpick: can we put `()` around `ovirt_obj`? `ovirt_attrs(ovirt_obj)` is a bit easier to read.